conf/modules.config

```diff
@@ -68,12 +68,18 @@ process {
     withName: FASTQC {
         ext.args = '--quiet'
         ext.when         = { !params.skip_qc }
+        publishDir = [
+            path: { "${params.outdir}/fastqc" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
     }
 
-    withName: FASTQC_FOR_TRIM {
+    withName: FASTQC_FOR_FASTP {
```

Comment:
why the name change?

Reply:
FASTQC_FOR_TRIM does not exist, it was related to a non existent module. The right modules is FASTQC_FOR_FASTP is the correct name